modules/core/src/main/java/org/apache/ignite/internal/processors/rest/GridRestProcessor.java

```diff
@@ -349,7 +351,7 @@
                             .a(", params=").a(tskReq.params());
                     }
 
-                    sb.a(", err=").a(e.getMessage() != null ? e.getMessage() : e.getClass().getName()).a(']');
+                    sb.a(", err=").a(getErrorMessage(e)).a(']');
```

Comment:
I think this can break Web Console, because it is also relay on this output.

Reply:
fixed